easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.2-2023b.yml

```
@@ -29,3 +29,19 @@ easyconfigs:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23155
         from-commit: b35081d4454f54996108088780f6554739dc4891
   - Ruby-3.4.2-GCCcore-13.2.0.eb
+  - Boost.Python-NumPy-1.83.0-gfbf-2023b.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23155
+        from-commit: b35081d4454f54996108088780f6554739dc4891
```

Hmm, this is from another PR. Though it works (your commit is newer), it's quite confusing, so I'd change it to the relevant PR/commit:
```suggestion
        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23893
        from-commit: 7f5e2d7d079e9c150d731b0525a310bc6b9008d4
```

---

Ah, I remember now: it was in my PR, but someone duplicated it in another PR, which got merged earlier... (so then I removed it)